arbitrum-docs/run-arbitrum-node/03-run-full-node.md

```diff
@@ -44,10 +44,12 @@ Even though there are alpha and beta versions of the <a data-quicklook-from='arb
 
 - Latest Docker Image: <code>@latestNitroNodeImage@</code>
 - Database snapshot (required for Arbitrum One, optional for other chains)
+  - Use the parameter `--init.latest <snapshot type>`, accepted values: "archive" | "pruned" | "genesis". (Only Arbitrum One now)
   - Use the parameter `--init.url` on first startup to initialize the Nitro database (you can find a list of snapshots [here](https://snapshot.arbitrum.foundation/index.html)). Example: <code>--init.url="@arbOneNitroPrunedSnapshot@"</code>.
```

Comment:
Should we remove this point? (just for clarity, they can see the different options in the page you linked at the end)